index.html

```diff
@@ -0,0 +1,129 @@
+                    <li class="#"><a href="#">Контакты</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <h1>Интернет магазин device</h1>
+            
+            <section class="slider">
+                <h2>Реклама промо товаров</h2>
+                <div class="slider_list">
+                    <div class="slider_item">
+                        <img src="#" alt="Самая длинная палка для селфи">
+                        <p>01</p>
+                        <p class="slider_item">Делай селфи, как Бен Стиллер!</p>
+                        <p class="slider_item">Самая длинная палка для селфи доступна в нашем магазине. Восемь (Восемь Карл!) метров и весом всего 5 килограмм.</p>
+                        <a class="slider_item" href="#">подробнее</a>
+                        <dl class="slider_item">
+                            <dt>Длина палки</dt>
+                            <dd>8,5 м</dd>
+                            <dt>Вес палки</dt>
+                            <dd>5 кг</dd>
+                            <dt>Материал</dt>
+                            <dd>Карбон</dd>
+                        </dl>            
+                    </div>
+                    <div class="slider_item">
+                        <img src="#" alt="фитнес-браслет">
+                        <p>02</p>
+                        <p class="slider_item">Худеем правильно!</p>
+                        <p class="slider_item">Мотивирующие фитнес-браслеты помогут найти в себе силы не пропускать занятия и соблюдать диету.</p>
+                        <a class="slider_item" href="#">подробнее</a>
+                        <dl class="slider_item">
+                            <dt>Без подзарядки</dt>
+                            <dd>48 часов</dd>                            
+                        </dl>            
+                    </div>
+                    <div class="slider_item">
+                        <img src="#" alt="квадрокоптер">
+                        <p>03</p>
+                        <p class="slider_item">Пархает как бабочка, жалит как пчела!</p>
+                        <p class="slider_item">Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+                        <a class="slider_item" href="#">подробнее</a>
+                        <dl class="slider_item">
+                            <dt>Дальность полета</dt>
+                            <dd>800 м</dd>
+                            <dt>Радиус поражения</dt>
+                            <dd>50 м</dd>                            
+                        </dl>            
+                    </div>
+                </div>                
+            </section>
+            <section>
+                <h2>Популярные товары</h2>
+                <div>
+                    <ul>
+                        <li><a href="#"><img src="#">Виртуальная реальность</a></li>
```

Comment:
here you will slightly change the markdown, but I won't spoiler right now, you'll figure out how to do it by yourself

Reply:
ok :)